src/binding.cc

```diff
@@ -684,81 +669,80 @@ class Client : public ObjectWrap {
           for (i = 0; i < vlen; ++i)
             new_buf[i] = buf[i];
-          field_value = String::New(new_buf, vlen);
+          field_value = NanNew<String>(new_buf, vlen);
           delete[] new_buf;
         } else
-          field_value = String::New(cur_query.row[f], lengths[f]);
+          field_value = NanNew<String>(cur_query.row[f], lengths[f]);
 
         if (cur_query.use_array)
           row->Set(f, field_value);
         else
-          row->Set(cur_query.column_names[f], field_value);
+          row->Set(NanNew<String>(cur_query.column_names[f]), field_value);
 
         if (config.metadata) {
           field = fields[f];
           if (cur_query.use_array) {
-            types->Set(f, String::New(FieldTypeToString(field.type)));
-            charsetNrs->Set(f, Integer::NewFromUnsigned(field.charsetnr));            
-            dbs->Set(f, String::New(field.db));
-            tables->Set(f, String::New(field.table));
-            orgTables->Set(f, String::New(field.org_table));
-            names->Set(f, String::New(field.name));
-            orgNames->Set(f, String::New(field.org_name));
+            types->Set(f, NanNew<String>(FieldTypeToString(field.type)));
+            charsetNrs->Set(f, NanNew<Integer, uint32_t>(field.charsetnr));            
+            dbs->Set(f, NanNew<String>(field.db));
+            tables->Set(f, NanNew<String>(field.table));
+            orgTables->Set(f, NanNew<String>(field.org_table));
+            names->Set(f, NanNew<String>(field.name));
+            orgNames->Set(f, NanNew<String>(field.org_name));
           }
           else {
-            types->Set(cur_query.column_names[f], String::New(FieldTypeToString(field.type)));
-            charsetNrs->Set(cur_query.column_names[f], Integer::NewFromUnsigned(field.charsetnr));            
-            dbs->Set(cur_query.column_names[f], String::New(field.db));
-            tables->Set(cur_query.column_names[f], String::New(field.table));
-            orgTables->Set(cur_query.column_names[f], String::New(field.org_table));
-            names->Set(cur_query.column_names[f], String::New(field.name));
-            orgNames->Set(cur_query.column_names[f], String::New(field.org_name));
+            types->Set(NanNew<String>(cur_query.column_names[f]), NanNew<String>(FieldTypeToString(field.type)));
+            charsetNrs->Set(NanNew<String>(cur_query.column_names[f]), NanNew<Integer, uint32_t>(field.charsetnr));            
+            dbs->Set(NanNew<String>(cur_query.column_names[f]), NanNew<String>(field.db));
+            tables->Set(NanNew<String>(cur_query.column_names[f]), NanNew<String>(field.table));
+            orgTables->Set(NanNew<String>(cur_query.column_names[f]), NanNew<String>(field.org_table));
+            names->Set(NanNew<String>(cur_query.column_names[f]), NanNew<String>(field.name));
+            orgNames->Set(NanNew<String>(cur_query.column_names[f]), NanNew<String>(field.org_name));
```

Comment:
These first `NanNew<String>()`s can be removed too since `column_names` should already contain new strings.